ansible/aws/start_migration.yaml

```diff
@@ -13,7 +13,7 @@
   - name: <== Executing payload ==>
     shell: "python3 /tmp/start.py {{ bucket }} {{ access_key }} {{ secret_key }} '{{ mongodb }}' {{ project }}"
     register: st
-    become: True
+    become: true
```

Comment:
This become is redundant because become is set to true for the whole play on the top